app/src/controllers/organizations/selectors.js

```diff
@@ -19,3 +19,5 @@ export const organizationsSelector = (state) => state.organizations || {};
 export const organizationsListSelector = (state) => organizationsSelector(state).list || [];
 
 export const organizationsListLoadingSelector = (state) => organizationsSelector(state).listLoading;
+
+export const organizationsProjectsSelector = (state) => organizationsSelector(state).projects;
```

Comment:
Can we store the projects in the `organization` controller?
I imagine the next structure
```
---constollers
------instance (holds organization list, its settings, etc.)
--------- {subController} (can be decomposed on more detailed structure by reducers combination to improve readability, like it was done in the `controllers/administrate` for its parts)
------organization (holds active organization, its settings, project list, org members, etc.)
--------- {subController} (members/projects, etc.)
```
What do you think?
cc @maria-hambardzumian, @Vadim73i

Reply:
Done.